package.json

```diff
@@ -7,16 +7,17 @@
   "author": "Michał Pierzchała <thymikee@gmail.com>",
   "license": "BSD-3",
   "dependencies": {
-    "jest-zone-patch": "^0.0.7",
-    "jsdom": "^11.3.0",
-    "ts-jest": "^21.0.0"
+    "jest-zone-patch": "^0.0.8",
+    "jsdom": "11.5.1",
+    "ts-jest": "21.2.4"
   },
   "devDependencies": {
-    "jest": "^21.0.1",
-    "typescript": "^2.3.0"
+    "jest": "21.2.1",
+    "typescript": "~2.4.2"
   },
   "peerDependencies": {
-    "jest": "^21.0.1"
+    "jest": "^21.0.1",
+    "zone.js": "^0.8.18"
```

Comment:
Can you remove this? `jest-zone-patch` already has a peer dep on Zone.js

Reply:
sure will do. Thank you :)